src/utils/linkDatabase.ts

```diff
@@ -0,0 +1,62 @@
+import { promises as fs } from "fs";
+import os from "os";
+import path from "path";
+
+async function getLinkContent(): Promise<Map<string, string[]>> {
+    const filePath = path.join(os.homedir(), ".geneziolinks");
```

Comment:
I'm worried that we are creating too many `.genezio(.*)` files. I propose to have a `.genezio` directory that stores all this helper files to avoid polluting the user home directory.